jobs/e2e_node/crio/crio.ign

```diff
@@ -24,6 +24,11 @@
         "contents": "[Unit]\nDescription=Download and install crio binaries and configurations.\nAfter=network-online.target\n\n[Service]\nType=oneshot\nExecStartPre=/usr/bin/bash -c '/usr/bin/curl --fail --retry 5 --retry-delay 3 --silent --show-error -o /usr/local/crio-nodee2e-installer.sh  https://raw.githubusercontent.com/cri-o/cri-o/3e371aa628d3283fd181590498684497628d324c/scripts/node_e2e_installer'\nExecStart=/usr/bin/bash /usr/local/crio-nodee2e-installer.sh\n\n[Install]\nWantedBy=multi-user.target\n",
         "enabled": true,
         "name": "crio-install.service"
+      },
+      {
+        "contents": "[Unit]\nDescription=Download and install dbus-tools.\nAfter=network-online.target\n\n[Service]\nType=oneshot\nExecStart=/usr/bin/rpm-ostree install --apply-live --allow-inactive dbus-tools\n\n[Install]\nWantedBy=multi-user.target\n",
```

Comment:
Do you want to do this `Before=crio-install.service` but `After=network-online.target`? This way we would make sure when crio comes up the debus-tools are already installed. 

As soon as crio is up and is detected by test-infra, it will initiate transfer of k8s binaries and start executing the node e2e tests. So in case we take longer to install dbus-tools package (e.g bad network), we could end up in a situation where the test that requires this package is executed without having this package actually present on the system.

Reply:
great point, updated